ffi/scripts/verify_golangci_yaml_changes.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
+
+curl -o /tmp/upstream.yml https://raw.githubusercontent.com/ava-labs/avalanchego/refs/heads/master/.golangci.yml
+    
+# Generate diff
+diff /tmp/upstream.yml .golangci.yaml > /tmp/diff.txt || true
+
+# Compare with expected diff
+if ! diff /tmp/diff.txt .golangci_yaml_expected_changes.txt; then
+    echo ".golangci.yaml has unexpected changes from AvalancheGo"
+    exit 1
```

Comment:
Not sure I'm a fan of this as it introduces a period of time where our CI _will_ fail whenever the upstream file changes regardless of the changes that were made. This will add unnecessary friction to PRs.

This script should only run when the `.golangci.yaml` in this repo is being updated so that only then we validate the changes against the expected diff.